tensorflow/python/eager/function.py

```diff
@@ -2166,27 +2171,9 @@ def _build_call_outputs(self, result):
     Returns:
       The actual call output.
     """
-    # TODO(jlchu): implement in C++.
-    if self._func_graph.structured_outputs is None:
-      return result
-
-    if result:
-      if self._ndarrays_list:
-        return [np_arrays.tensor_to_ndarray(o) for o in result]
-      elif self._ndarray_singleton:
-        return np_arrays.tensor_to_ndarray(result[0])
-
-    # Replace outputs with results, skipping over any 'None' values.
-    outputs_list = nest.flatten(
-        self._func_graph.structured_outputs, expand_composites=True)
-    j = 0
-    for i, o in enumerate(outputs_list):
-      if o is not None:
-        outputs_list[i] = result[j]
-        j += 1
-    ret = nest.pack_sequence_as(self._func_graph.structured_outputs,
-                                outputs_list, expand_composites=True)
-    return ret
+    return self._cpp_concrete_function._build_call_outputs(
```

Comment:
per discussion, since the performance is not getting better with c++ code, please keep the python code, and revert changes here. Instead add the comment to use c++ code later once performance is getting better

Reply:
Reverted. Currently this PR just adds the C++ file and appropriate dependencies in the BUILD file, but the code within is not being called.